.github/workflows/prepare-for-minor-release.yml

```
@@ -144,7 +144,7 @@ jobs:
     name: Update `${{ matrix.previous_version }}` release branch.
     strategy:
       matrix:
-        previous_version: [2.7.x, 2.6.x, 2.5.x, 2.4.x, 2.3.x, 2.2.x, 2.1.x, 2.0.x]
+        previous_version: [2.8.x, 2.7.x, 2.6.x, 2.5.x, 2.4.x, 2.3.x, 2.2.x, 2.1.x, 2.0.x]
```

Might as well add 2.9.x here now, too so it's ready for next time.
```suggestion
        previous_version: [2.9.x, 2.8.x, 2.7.x, 2.6.x, 2.5.x, 2.4.x, 2.3.x, 2.2.x, 2.1.x, 2.0.x]
```